cmd/executor/cmd/root.go

```diff
@@ -23,23 +23,21 @@ import (
 
 	"github.com/genuinetools/amicontained/container"
 
-	"github.com/GoogleContainerTools/kaniko/pkg/executor"
-
 	"github.com/GoogleContainerTools/kaniko/pkg/constants"
+	"github.com/GoogleContainerTools/kaniko/pkg/executor"
 	"github.com/GoogleContainerTools/kaniko/pkg/util"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
 var (
-	dockerfilePath              string
-	destination                 string
-	srcContext                  string
-	snapshotMode                string
-	bucket                      string
-	dockerInsecureSkipTLSVerify bool
-	logLevel                    string
-	force                       bool
+	dockerfilePath string
+	destination    string
+	srcContext     string
+	snapshotMode   string
+	bucket         string
+	logLevel       string
```

Comment:
looks like you dropped the dockerinsecuretlsverify in a merge conflict.

Reply:
yah, it didn't look like it was doing anything now that we switched to go-containerregistry

Reply:
(probably should have done that in a different commit)